docs/admin/install.rst

```diff
@@ -1086,33 +1097,31 @@ Running maintenance tasks
 +++++++++++++++++++++++++
 
 For optimal performance, it is good idea to run some maintenance tasks in the
-background. This is now automatically done by :ref:`celery` and covers following tasks:
+background.
+This is now automatically done by :ref:`celery` and covers the following tasks:
 
 * Configuration health check (hourly).
 * Committing pending changes (hourly), see :ref:`lazy-commit` and :djadmin:`commit_pending`.
 * Updating component alerts (daily).
 * Update remote branches (nightly), see :setting:`AUTO_UPDATE`.
-* Translation memory backup to JSON (daily), see :djadmin:`dump_memory`.
-* Fulltext and database maintenance tasks (daily and weekly tasks), see :djadmin:`cleanuptrans`.
+* Translation-memory backup to JSON (daily), see :djadmin:`dump_memory`.
+* Fulltext- and database-maintenance tasks (daily and weekly tasks), see :djadmin:`cleanuptrans`.
 
 .. versionchanged:: 3.2
 
-   Since version 3.2, the default way of executing these tasks is using Celery
-   and Weblate already comes with proper configuration, see :ref:`celery`.
+   Since version 3.2, the default way of executing these tasks is by using Celery,
+   and Weblate already comes with proper configuration for it, see :ref:`celery`.
 
 .. _production-encoding:
 
 System locales and encoding
 +++++++++++++++++++++++++++
 
-The system locales should be configured to UTF-8 capable ones. On most Linux
-distributions this is the default setting. In case it is not the case on your
-system, please change locales to UTF-8 variant.
+If for some reason the system locales for your distribution are not
+configured to UTF-8 capable ones, you can do so by
+editing :file:`/etc/default/locale` and setting ``LANG="C.UTF-8"`` there.
```

Comment:
This seems to suggest C.UTF-8 is UTF-8, when really it changes the locale that could have been UTF-8 all along(?).

Reply:
`C` locale is not UTF-8.

Reply:
Right, however if I have "LANG="nb_NO.UTF-8"" I don't need "C.UTF-8", which I could be lead to believe is the only valid (or a better) option.

Reply:
Yes, the original docs said that for a reason:

> The system locales should be configured to UTF-8 capable ones. On most Linux distributions this is the default setting.

Reply:
Evident from "if for some reason", but I guess.
MOVE